docs/wiki/get-started/IO-with-Adapters.md

```diff
@@ -0,0 +1,65 @@
+In [First Steps](First-Steps) and [More with csp](More-with-CSP) you created example/sample data for the streaming workflows. In real workflows, you use data stored in particular formats and storage spaces, that can be accessed directly or through an API.
+
+csp has [several built-in "adapters" to access certain types of data](Input-Output-Adapters-API), including Kafka and Parquet. csp requires friendly (Time Series) data types, and the I/O adapters form the interface between the data types. You can also write your own adapters for other data types, check the corresponding how-to guides for more details.
+
+In this tutorial, you and write to and read from Parquet files on the local file system.
+
+csp has the `ParquetWriter` and `ParquetReader` adapters to stream data to and from Parquet files. Check out the complete [API in the Reference documentation](https://github.com/Point72/csp/wiki/Input-Output-Adapters-API#parquet).
+
+> \[!NOTE\]
+> csp can handle historical and real-time data, and the csp program remains similar in both cases.
+
+## Example
+
+You start with an Example `csp.Struct` data type which will be streamed into a Parquet file.
+
+```python
+from csp.adapters.parquet import ParquetOutputConfig, ParquetWriter, ParquetReader
+
+class Example(csp.Struct):
+    int_val: int
+    float_val: float
+```
+
+## Write to a Parquet file
+
+In a graph, create some sample values for `Example` and use `ParquetWriter` to steam it to a Parquet file.
+
+1. The `timestamp_column_name` is how csp preserves the timestamps. Since you need to read this file back into csp, you can provide a column name. If this was the final output and the time stamp information is not required, you can provide `None`.
+
+1. You can provide optional configurations to `clearconfig` in the `ParquetOutputConfig` format (which can set `allow_overwrite`, , `batch_size`, `compression`, `write_arrow_binary`).
```

Comment:
not sure where `clearconfig` came from, the arg should be `config`
please make sure the example code in these docs run as expected

Reply:
Yes, that's a typo. Also verified executing these.